README.md

```
@@ -12,7 +12,7 @@ This project is tested with BrowserStack.
 
 <a href="https://flatt.tech/oss/gmo/trampoline" target="_blank"><img src="https://flatt.tech/assets/images/badges/gmo-oss.svg" height="20px"/></a>
 
-<img src="./docs/src/assets/stlite.svg" style="background: white;" >
+<img src="./docs/src/assets/logo-themed.svg" >
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)